src/components/report/VariableControls.js

```diff
@@ -1,17 +1,11 @@
 export default {
 	props: ["font", "showAxes", "showInstances", "showTitles", "showStyles"],
-	data() {
-		return {
-			activeInstance: ""
-		};
-	},
+	data: self => ({
```

Comment:
I think you can also use `this` in the data function, if you declare it like this:

```js
data() {
  return {
    activeInstance: "",
    axes: self.font.variables.axes,
    instances: self.font.variables.instances
  }
}
```

Reply:
True, I chose this approach to keep in line with the other "data as arrow functions" declarations. Kinda makes it feel consistent, but I'm not married to this approach. Do you prefer the data/return approach?

Reply:
I'm not sure which style is more typical in Vue. The Vue docs do warn to not use arrow functions in some cases so I think not using arrow functions is a better default.

![Screenshot 2020-08-26 at 09 22 56](https://user-images.githubusercontent.com/304603/91273579-c1687c80-e77d-11ea-8f81-5964df97530b.png)


https://vuejs.org/v2/guide/instance.html#Data-and-Methods

Reply:
Sounds reasonable then! 😄 I'll pick this up in the generic clean up PR I'm working on now.